appdatabase/migrations/sql/1722404163_add_status_proxy_urls_to_networks.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE networks ADD COLUMN proxy_rpc_url VARCHAR NOT NULL DEFAULT "";
```

Comment:
Why do we need to add new entries?
From status go point of view they are simply different url and are the default one